wrappers/services.go

```diff
@@ -532,6 +537,9 @@ KillMode={{.KillMode}}
 {{- if .KillSignal}}
 KillSignal={{.KillSignal}}
 {{- end}}
+{{- if .VitalityScore }}
+OOMScoreAdjust=-{{.VitalityScore}}
```

Comment:
should we have the minus here? or pass in a negative integer but see also naming wondering below